### ISSUE TITLE
Check migration to spring boot 1.3.x

### ISSUE THREAD

**Issue by gioppoluca** [CloudOpting/cloudopting-manager#113]
We need to check if migrating to the latest version of the spring boot env causes problems in the project.
If not we need to stay on the latest before going final on the orchestrator to get the benefit of the bug fixing and enhancement there are

**Comment by gioppoluca** [comment]
For @ciprianpavel I assign it to you but I'd like Daniel to take time to look after this since I think he has the better understanding of the whole implications of this change.

**Comment by cpopsa** [comment]
Changing spring boot version from 1.2.3.RELEASE (current used) to 1.3.1.RELEASE (latest release) implies changing also the version of the following dependencies:
- Spring security from 3.2.7.RELEASE to 4.0.3.RELEASE
	Using 4.0.3.RELEASE fails with compilation error in SecurityConfiguration line 109 - method authorizeRequests() not found

- Jackson from 2.4.5 to 2.6.4
	Using 2.6.4 fails with compilation error in JacksonConfiguration - JacksonJodaFormat not found
	
- Snakeyaml from 1.14 to 1.16
	Using 1.16 fails at server startup with: found character '\t(TAB)' that cannot start any token. (Do not use \t(TAB) for indentation) in 'reader', line 7, column 1:
	
- Spring framework from 4.1.6.RELEASE to 4.2.4.RELEASE
	This also could result in possible errors
	
In conclusion, migration from spring boot 1.2.3.RELEASE to 1.3.1.RELEASE is not feasible now.

**Comment by gioppoluca** [comment]
Can you check the latest 1.2.8 also, at least we could get the benefits of bug fixing in this.

**Comment by cpopsa** [comment]
1.2.8.RELEASE seems to be ok. I'v changed the version, from 1.2.3.RELEASE to 1.2.8.RELEASE
